ext/opentelemetry-ext-dbapi/src/opentelemetry/ext/dbapi/__init__.py

```
@@ -0,0 +1,117 @@
+        kwargs: typing.Dict[any, any],
+    ):
+        """Patch cursor instance in a specific connection.
+        """
+        cursor = wrapped(*args, **kwargs)
+        for func in QUERY_WRAP_METHODS:
+            wrapt.wrap_function_wrapper(cursor, func, self.add_span)
+        return cursor
+
+    # pylint: disable=unused-argument
+    def add_span(
+        self,
+        wrapped: typing.Callable[..., any],
+        instance: typing.Any,
+        args: typing.Tuple[any, any],
+        kwargs: typing.Dict[any, any],
+    ):
+        """Patch execute, executeMany and callproc methods in cursor and create span.
+        """
+        name = self._database_component
+        database = self._connection_props.get("database", "")
+        if database:
+            name += "." + database
+        query = args[0] if args else ""
+        # Query with parameters
+        if len(args) > 1:
+            query += " params=" + str(args[1])
+
+        with self._tracer.start_as_current_span(
+            name, kind=SpanKind.CLIENT
+        ) as span:
+            span.set_attribute("component", self._database_component)
+            span.set_attribute("db.type", self._database_type)
+            span.set_attribute("db.instance", database)
+            span.set_attribute("db.statement", query)
+            span.set_attribute(
+                "db.user", self._connection_props.get("user", "")
+            )
+            span.set_attribute(
+                "peer.hostname", self._connection_props.get("host", "")
+            )
+            port = self._connection_props.get("port")
+            if port is not None:
+                span.set_attribute("peer.port", port)
+
+            try:
+                result = wrapped(*args, **kwargs)
+                span.set_status(Status(StatusCanonicalCode.OK))
+                return result
+            except Exception as ex:  # pylint: disable=broad-except
+                span.set_status(Status(StatusCanonicalCode.UNKNOWN, str(ex)))
```

You want to re-raise the exception here (using bare `raise`), not swallow it.